src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
 // write your code here
+const promise1 = new Promise((resolve) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+function ifSuccess(successMessage) {
+  const div = document.createElement('div');
+
+  div.classList.add('message');
+  div.textContent = successMessage;
+  document.body.appendChild(div);
+}
```

Comment:
The function `ifSuccess` is not named according to the convention. Function names should usually start with a verb, as they perform actions. A more appropriate name could be `handleSuccess` or `displaySuccessMessage`.